baseplate/thrift/message_queue.thrift

```diff
@@ -0,0 +1,35 @@
+namespace go baseplate.remote_message_queue
+namespace py baseplate_remote_message_queue
+namespace java com.reddit.baseplate.remote_message_queue
+
+struct PutRequest {
+    1: binary data;
+}
+
+struct PutResponse {}
+
+
+struct GetRequest {
+    1: binary key;
```

Comment:
What is the key used for? AFAIU this is a simple FIFO queue, so it just needs to `put(data)` at the back of the queue and `get()` from the front of the queue?